### DISCUSSION TITLE
Notes Jul 25th, 2022

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#73] [Governing Board Meetings]
On Monday, July 25th, 2022 we had a Governing Board meeting and we mostly discussed the upcoming election and the draft of the MapLibre Charter. The Charter draft will be published later this week as a pull request to maplibre/maplibre where everyone is welcome to provide feedback.

Feel free to ask any question!

## Attendees
@wipfli
@klokan
@nyurik
@lseelenbinder

## Charter
@klokan will follow up on charter review offline until Wednesday July 27th, 2022. After that, the Charter will be published as pull request to maplibre/maplibre for broader community feedback.
@nyurik: Ranked vote: Voting Members say who their first, second, third most preferred candidate is. This could work better than same-weight-vote for small groups. Agreed that this is the way to go.
@lseelenbinder: GitHub Polls are not well suited for our needs. Looking into other solutions which will probably use email. Maybe opavote.com.

## Advisory Council Meeting
The newly formed Advisory Council acts as a forum for feedback and exchange between participants of the MapLibre Sponsorship Program and MapLibre. The Council will meet on a regular basis and the members of the MapLibre Governing Board and the Technical Steering Committee can join the meetings. 

@wipfli We will publish the details of the MapLibre Sponsorship Program before the election of the Governing Board.
@wipfli is scheduling a first Advisory Council meeting for the time after the Governing Board election.
Use Governing Board email address to have an email track record with third-parties.

## Domain
@nyurik: Can the domain maplibre.org be transferred to the MapLibre Governing Board?
@klokan will create a Cloudflare account and give access to the Governing Board.
Will use OpenCollective virtual cards to pay the domain. @nyurik will open one until Wednesday July 27th, 2022.

## Trademark
@nyurik: Asked Open Source Collective if we can register the trademark "MapLibre".
If we don't like Open Source Collective, how can we switch?

**Comment by maxammann** [comment]
@klokan I already created a cloudflare account for maplibre and invited somebody from th governing board. Should we use that instead?

**Reply by wipfli** [reply]
That's a good idea. Can you make a pull request with some details about the usage of the cloudflare account to SECRETS.md?

**Comment by wipfli** [comment]
https://github.com/maplibre/maplibre/blob/main/SECRETS.md
